components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,215 @@
+ */
+
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { adminMenu } from "./admin-menu";
+
+import { LicenseContext } from "../license-context";
+import { ReactElement, useContext, useEffect } from "react";
+import { getGitpodService } from "../service/service";
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/check-circle.svg";
+import { LicenseInfo } from "@gitpod/gitpod-protocol";
+import { ReactComponent as XSvg } from "../images/x.svg";
+import { ReactComponent as CheckSvg } from "../images/check.svg";
+import SolidCard from "../components/SolidCard";
+import Card from "../components/Card";
+
+export default function License() {
+    const { license, setLicense } = useContext(LicenseContext);
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, statusMessage] = license ? getSubscriptionLevel(license) : defaultMessage();
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card>
+                        {licenseLevel}
+                        {paid}
+                        <div className="mt-4 font-semibold">Available features:</div>
+                        <div className="flex flex-col items-start">
+                            {features &&
+                                features.map((feat: string) => (
+                                    <span className="inline-flex space-x-1">
+                                        {featureList?.includes(feat) ? (
+                                            <CheckSvg fill="currentColor" className="self-center" />
+                                        ) : (
+                                            <XSvg fill="currentColor" className="self-center h-2" />
+                                        )}
+                                        <span>{capitalizeInitials(feat)}</span>
+                                    </span>
+                                ))}
+                        </div>
+                    </Card>
+                    <SolidCard>
+                        <div className="text-gray-600 dark:text-gray-200 py-4 flex-row flex font-semibold items-center">
```

Comment:
**nitpick**: Moving this back would be great! Sorry if the suggestion in https://github.com/gitpod-io/gitpod/pull/9343#discussion_r853460822 was confusing, @nandajavarma! I meant to keep the top margin and padding as is but move the button after the text elements.

```suggestion
                        <div className="text-gray-600 dark:text-gray-200 py-2 flex-row flex font-semibold items-center">
```